django_irods

```diff
@@ -1 +1 @@
-Subproject commit 7a659203f8d548ad5e10f6a762bc601ad068ebbc
+Subproject commit f51d0b1c52b14e4c5759f33b3ac6531f34dcf82b
```

Comment:
Thank you, I'll fix that.

For the record, git submodules are notoriously finicky and redundant for most uses in a Django project with a proper requirements.txt anyway.